blocks/page_list/templates/itinerary/Itinerary.jsx

```diff
@@ -25,7 +29,7 @@ export default class Itinerary extends React.Component {
 
   constructor(...args) {
     super(...args);
-    this.state = props.itinerary || getItinerary();
+    this.state = this.props.itinerary || getItinerary();
```

Comment:
for ...args - we are spreading all other arguments that are passed in. More so future proofing?